src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface FigureArea {
```

Comment:
```suggestion
interface AreaCalculator {
```
Remember Abstraction. Not only figure can have area.